### ISSUE TITLE
publish for Scala 2.13.0

### ISSUE THREAD

**Issue by SethTisue** [playframework/omnidoc#24]
previously: https://github.com/playframework/omnidoc/issues/20

**Comment by dgouyette** [comment]
That would be useful. I would like to a few tests play 2.7 and scala 2.13.M5 and i got the following exception : 

```
[info] Resolving com.typesafe.play#play-omnidoc_2.13.0-M5;2.7.0 ...
[warn] 	module not found: com.typesafe.play#play-omnidoc_2.13.0-M5;2.7.0
[warn] ==== local: tried
[warn]   /Users/damiengouyette/.ivy2/local/com.typesafe.play/play-omnidoc_2.13.0-M5/2.7.0/ivys/ivy.xml
[warn] ==== public: tried
[warn]   https://repo1.maven.org/maven2/com/typesafe/play/play-omnidoc_2.13.0-M5/2.7.0/play-omnidoc_2.13.0-M5-2.7.0.pom
[warn] ==== local-preloaded-ivy: tried
[warn]   /Users/damiengouyette/.sbt/preloaded/com.typesafe.play/play-omnidoc_2.13.0-M5/2.7.0/ivys/ivy.xml
[warn] ==== local-preloaded: tried
[warn]   file:////Users/damiengouyette/.sbt/preloaded/com/typesafe/play/play-omnidoc_2.13.0-M5/2.7.0/play-omnidoc_2.13.0-M5-2.7.0.pom
[info] Resolving jline#jline;2.14.6 ...
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: com.typesafe.play#play-omnidoc_2.13.0-M5;2.7.0: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn]
[warn] 	Note: Unresolved dependencies path:
[warn] 		com.typesafe.play:play-omnidoc_2.13.0-M5:2.7.0 ((play.sbt.PlaySettings) PlaySettings.scala#L84)
[warn] 		  +- org.cestpasdur:play-git-info-webapp_2.13.0-M5:0.2.1
```

**Comment by mkurz** [comment]
@dgouyette As a workaround for now you can set
```
PlayKeys.playOmnidoc := false
```
in your `build.sbt`

**Comment by dgouyette** [comment]
Thanks @mkurz i'll do that

**Comment by marcospereira** [comment]
Update to publish for Scala 2.13.0 GA. :-)

**Comment by ignasi35** [comment]
Fixed in https://github.com/playframework/omnidoc/pull/51 (Play `2.7.3` is the first in `2.7` series to be released with `2.13` support).
